README.md

```diff
@@ -22,109 +22,89 @@ Azure Thunderbolt is designed to run batch Spark jobs that require additional on
 ```bash
     pip install -e .
 ```
-4. Rename 'secrets.cfg.template' to 'secrets.cfg' and fill in the fields for your Batch account and Storage account.
+4. Initialize the project in a directory [This will automatically create a *.thunderbolt* folder with config files in your working directory]:
+```bash
+    azb spark init
+```
+5. Fill in the fields for your Batch account and Storage account in your *.thunderbolt/secrets.yaml* file. (We'd also recommend that you enter SSH key info in this file)
 
    Thunerbolt is built on top of two core Azure services, [Azure Batch](https://azure.microsoft.com/en-us/services/batch/) and [Azure Storage](https://azure.microsoft.com/en-us/services/storage/). Create those resources via the portal (see [Getting Started](./docs/00-getting-started.md)).
 
 ## Quickstart Guide
 
-The entire experience of this package is centered around a few commands.
+The core experience of this package is centered around a few commands.
+
+```sh
+# create your cluster
+azb spark cluster create
+```
+```sh
+# monitor and manage your clusters
+azb spark cluster get
+azb spark cluster list
+azb spark cluster delete
+```
+```sh
+# interact with your cluster
+azb spark cluster ssh
+azb spark cluster submit
+```
 
 ### Create and setup your cluster
 
 First, create your cluster:
 ```bash
 azb spark cluster create \
-    --id <my-cluster-id> \
-    --size <number of nodes> \
-    --vm-size <vm-size> \
-    --custom-script <path to custom bash script to run on each node> (optional) \
-    --wait/--no-wait (optional)
+    --id <my_cluster_id> \
+    --size <number_of_nodes> \
+    --vm-size <vm_size>
 ```
 
-You can also create your cluster with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using **--size-low-pri** instead of **--size**:
+You can also create your cluster with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using `--size-low-pri` instead of `--size`:
 ```
 azb spark cluster create \
-    --id <my-cluster-id> \
-    --size-low-pri <number of low-pri nodes> \
-    --vm-size <vm-size>
+    --id <my_cluster_id> \
+    --size-low-pri <number_of_low-pri_nodes> \
```

Comment:
This _may_ break if the default config file has 
```yaml
size: 2
```
Since we do not allow mixed clusters. May want to update this to 
```bash
azb spark cluster create\
  --id <my_cluster_id> \
  --size 0
  --size-low-pri <number_of_low_pri_nodes>\
  --vm-size: <vm_size>
```

Reply:
This feels like a bug to me. While we do not support mixed nodes, we should override `--size` when `--size-low-pri` is used a param.

In the meantime, going with your suggestion.